### PR TITLE
Finalised Token Service fixes and improvement

### PR DESCRIPTION
**Pull request by victorskl**
* Removed policy sid as advice
* Tuned policy based on Policy Sim
* Tuned cognitor implementation
* Deprecated module helper.py
* Fixed and improved module impl rotate_service_jwt.py
* Fixed and improved module impl rotate_service_user.py
* Improved README.md

### CODE REVIEW

File: lib/workload/stateful/token_service/token_service/cognitor/__init__.py [hunk 1]
```diff
@@ -22,8 +22,14 @@ def __init__(self, user_pool_id: str, user_pool_app_client_id: str):
...
     def generate_password(length: int = 32) -> str:
+        """
+        Must meet the Cognito password requirements policy
+        https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-policies.html
+        """
+        if length < 8:
+            raise ValueError('Length must be at least 8 characters or more better')
         return ''.join(
-            random.SystemRandom().choice(string.ascii_letters + string.digits + '!@#$%&.') for _ in range(length)
+            random.SystemRandom().choice(string.ascii_letters + string.digits + '_-!@#%&.') for _ in range(length)
```
Comment (reisingerf) [inline lib/workload/stateful/token_service/token_service/cognitor/__init__.py:32]: Ah, user pool password policy demands at least one special character.... right!

**Review by reisingerf — APPROVED** [review]
LGTM

File: lib/workload/stateful/token_service/token_service/cognitor/__init__.py (hunk 1 above)
Comment (victorskl) [reply lib/workload/stateful/token_service/token_service/cognitor/__init__.py:32]: Yup. We can tune that though. But, let better be some symbols there, Flo.

Comment (reisingerf) [reply lib/workload/stateful/token_service/token_service/cognitor/__init__.py:32]: Yup, noticed the custom policy option. Agree though, better to stick to the default

**Comment by victorskl** [comment]
Merging...

**Pull request merged**